roles/nvidia-dgx-firmware/README.md

```diff
@@ -0,0 +1,61 @@
+# nvidia-dgx-firmware role
+
+This role makes use of the NVIDIA DGX firmware container and can be used to:
+1. Collect health & configuration diagnostics
+2. Update the firmware
+
+## Setup
+
+1) Download the latest DGX firmware container and put it in `./deepops/config/containers/dgx-firmware`. Keep the original file name. Update the role variables to reflect the version being used. 
+
+```yml
+# The Docker repo name
+firmware_update_repo: nvfw-dgx1
+
+# The Docker tag
+firmware_update_tag: 19.10.7
+```
+
+2) Change the `nv_mgmt_interface` variable to reflect the systems being collected from.
+
+```yml
+# The OS/mgmt interface on the server
+nv_mgmt_interface: enp1s0f0 # DGX-1
+# nv_mgmt_interface: enp134s0f0 # DGX-2
+# nv_mgmt_interface: enp2s0f1 # DGX-Station
+```
```

Comment:
Can we remove this? I can't find where it's used

Reply:
Used in [roles/nvidia-dgx-firmware/tasks/get-data.yml](https://github.com/NVIDIA/deepops/blob/11663d96ce6b838e116fabaeb8568ffbadbc71b0/roles/nvidia-dgx-firmware/tasks/get-data.yml), but I guess `get-data` is never used? @supertetelman - do you know why this is here?

Reply:
This was a special task that isn't included in the role by default. It's excluded because it is datacenter configuration specific and requires minor tweaking based on the networking layout.

This was to map hostnames, IPs, and MACs.